docs/src/whatsnew/3.1.rst

```diff
@@ -1,7 +1,7 @@
 .. include:: ../common_links.inc
 
-|iris_version| |build_date|
-***************************
+v3.1 (17 Aug 2021) [unreleased]
+*******************************
```

Comment:
I notice for the Iris 3 rc we included a date:
https://scitools-iris.readthedocs.io/en/v3.0.0rc0/whatsnew/3.0.html

Reply:
Sorry wasn't clear. I meant to say _just_ a date no `unreleased`

Reply:
Yes, I think that's effectively a _preliminary_ date -- the date of the RC.
Which we would change again we we make it a full release
(imho)

Reply:
I've done the same, but left in the '[unreleased]' tag, because I thought that made sense..

Reply:
You might also consider that it makes sense to label this with the 'rc0', 
but, again, we haven't done so in the past